src/features/collapsible-sidebar/__tests__/CollapsibleSidebarLogo.test.js

```diff
@@ -69,4 +69,13 @@ describe('components/core/collapsible-sidebar/CollapsibleSidebar', () => {
         const componentProp = sidebar.find('CollapsibleSidebarItem');
         expect(componentProp.find('LinkBase').prop('someprop')).toBe(someValue);
     });
+
+    test.each([true, false])('render Box logo when hideBoxLogo=%s', hideBoxLogo => {
+        const sidebar = getWrapper({
+            expanded: true,
+            hideBoxLogo,
+        });
+
+        expect(sidebar).toMatchSnapshot();
```

Comment:
this snapshot looks expensive. are we able to just check for the existence of `bdl-CollapsibleSidebar-logoLink`?

Reply:
Yes, I adjusted the tests.